redis.conf

```diff
@@ -1762,3 +1762,29 @@ rdb-save-incremental-fsync yes
 # Maximum number of set/hash/zset/list fields that will be processed from
 # the main dictionary scan
 # active-defrag-max-scan-fields 1000
+
+############################### PMEM CONFIG ###############################
+
+# Redis supports four different Memory Allocation Policies:
+#
+# only-dram: use only DRAM - do not use Persistent Memory
+# only-pmem: use only Persistent Memory - do not use DRAM
+# mixed-threshold: use Persistent Memory and DRAM - use threshold described by pmem-threshold
+# mixed-ratio: use Persistent Memory and DRAM - use ratio described by dram-pmem-ratio
+memory-alloc-policy only-dram
```

Comment:
Please update info: what happens if this directive is not defined i.e.: only DRAM will be used.

Reply:
agree, we use the directive way in the comments.